packages/metrics-server/README.md

```diff
@@ -0,0 +1,31 @@
+# Metrics Server Zarf Package
+
+This directory contains the Zarf package for the eks-addons Metrics Server Helm chart.
+
+## Overview
+
+The Metrics Server is a scalable, efficient source of container resource metrics for Kubernetes built-in autoscaling pipelines.
+
+## Package Contents
+
+This package includes:
+
+- `zarf.yaml`: The main configuration file for the Zarf package.
+- `zarf-config.yaml`: Additional configuration for the Zarf package.
+- `values/values.yaml`: The Helm chart values file for the Metrics Server Helm chart. These are default values that can be configured at package create time and deploy time.
+- `values/values-override.yaml`: Additional values to override the Helm chart values file. These settings are configurable and can be changed at deploy time. These settings are last in the helm chart values file and will override any settings set previously in other values files.
```

Comment:
does values/values-override.yaml exist?
also, should this be clarified to specify to modify 'deploy-values-override.yaml' instead?